client/src/components/LoginSession/LoginSession.js

```diff
@@ -1,18 +1,18 @@
 /* global window */
```

Comment:
I just noticed the js files in this module uses both 2 and 4 space indents.  I've raised a [separate issue](https://github.com/silverstripe/silverstripe-session-manager/issues/44) for this, rather than refactoring this as part of this PR which will cause a giant diff